intel_extension_for_transformers/transformers/utils/config.py

```diff
@@ -1054,8 +1054,8 @@ def __init__(
         sym: bool = True,
         lr: float = 0.0025,
```

Comment:
please set lr and minmax_lr value to None by default,  so that both will be calculated based on iters.

Reply:
please sync bits, group_size, sym default value with autoround too.

Reply:
improved